src/about/test/about.component.spec.ts

```diff
@@ -14,24 +14,29 @@ import {
 } from '../../test/';
 import { AboutComponent } from '../about.component';
 import { AboutService } from '../about.service';
+import { BehaviorSubject } from 'rxjs/BehaviorSubject';
+import { ActivatedRoute } from '@angular/router';
 
 describe('AboutComponent', () => {
   let componentInstance: AboutComponent;
   let componentFixture: ComponentFixture<AboutComponent>;
   let debugElement: DebugElement;
   let nativeElement: HTMLElement;
 
-  class AboutServiceMock {};
+  class ActivatedRouteMock {
+    params = new BehaviorSubject({jump: '123'});
+  }
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      imports: [ HttpModule ],
-      declarations: [ AboutComponent ],
+      declarations: [AboutComponent],
+      schemas: [NO_ERRORS_SCHEMA],
       providers: [
-        { provide: AboutService, useClass: AboutServiceMock },
-        { provide: LoaderService, useClass: LoaderServiceMock },
-        { provide: TitleHeaderService, useClass: TitleHeaderServiceMock },
-        { provide: LanguageService, useClass: LanguageServiceMock }
+        {provide: AboutService, useValue: {}},
+        {provide: LoaderService, useClass: LoaderServiceMock},
+        {provide: TitleHeaderService, useClass: TitleHeaderServiceMock},
```

Comment:
You use mocks for the same service in different files, better to create module with already mocked services an import only it. It will make your code more clear